app/decorators/models/solidus_address_name/order_decorator.rb

```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
```

Comment:
Do you mind fixing up this commit with the other one? In that way, the history will be more clear.